src/main/java/com/google/cloud/discotoproto3converter/proto3/DocumentToProtoConverter.java

```diff
@@ -352,11 +352,13 @@ private void readResources(Document document) {
 
         if (method.request() != null) {
           Message request = allMessages.get(method.request().reference());
-          String requestFieldName =
-              Name.anyCamel(request.getName(), "resource").toLowerUnderscore();
-          input.getFields().add(new Field(requestFieldName, request, false, null, null));
-          methodHttpOption.getProperties().put("body", requestFieldName);
-          methodSignatureParamNames.put("", requestFieldName);
+          if (request != null) {
```

Comment:
car request be null? if yes, under which circumstances? (it was not required for the whole `compute.proto` before so I'm wondering why it is needed now

Reply:
I received a null pointer exception for the first time today when using the ignorelist, and I thought this fixed it but I'm still having an issue, so I'll revert this change and submit in a different PR

Reply:
(reverted)